### PR TITLE
Fixes for title and block being null

### PR DESCRIPTION
**Pull request by sergeyf**

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 setuptools.setup(
     name="s2apler",
-    version="0.1",
+    version="0.11",
```
Comment (stefanc-ai2) [inline setup.py]: Should be 0.1.1?

**Comment by stefanc-ai2** [comment]
What's the output when sending the example request here https://github.com/allenai/scholar/issues/36053#issue-1640275292? An empty response?

**Comment by sergeyf** [comment]
I think it'll be a cluster with a single member: 1726032497.

**Comment by stefanc-ai2** [comment]
> I think it'll be a cluster with a single member: 1726032497.

Oh right. Ok, cool!

**Review by stefanc-ai2 — APPROVED** [review]

**Pull request merged**